tests.nix

```diff
@@ -0,0 +1,14 @@
+{ packages, system, pkgs }:
+if pkgs.lib.strings.hasSuffix "darwin" system
+then { }
+else {
+  hermes-module-test = (import ./modules/tests/hermes-test.nix) {
+    inherit (packages) hermes;
+    inherit system pkgs;
+    gaia = packages.gaia5;
+  };
+  gaia-module-test = (import ./modules/tests/gaia-test.nix) {
+    inherit system pkgs;
+    gaia = packages.gaia6;
+  };
```

Comment:
The idea is to aggregate tests here, so that we can merge them into `packages` and `checks` in order to make them easily available through the flakes interface.